modules/c++/sys/source/OSUnix.cpp

```diff
@@ -248,6 +248,16 @@ void sys::OSUnix::setEnv(const std::string& var,
     }
 }
 
+void sys::OSUnix::unsetEnv(const std::string& var)
+{
+    int ret = unsetenv(var.c_str());
+    // by definition, unsetenv does not consider a missing environment variable
+    // to be an error condition, so this should only throw if the environment
+    // variable could not be changed
+    if(ret == -1)
```

Comment:
Space in here: `if (ret == -1)`

Then below this, always use curly braces even if there's only a single line.  And always four space indents.  So...

```
if (ret == -1)
{
    throw ...
}
```